src/xmipp/libraries/reconstruction/volume_subtraction.cpp

```diff
@@ -269,21 +369,24 @@ class ProgVolumeSubtraction: public XmippProgram
 
     	if (sub==true)
     	{
-        	if (!fnVol1F.isEmpty() && !fnVol2A.isEmpty())
+        	if (Vol1F)
```

Comment:
This doesn't make sense. Either user passed `fnVol1F` and therefor wants it to be saved, or not

Reply:
The user may want to save the intermediate file or not (check boolean). If saved, it can be saved with default name (thus, fnVol1F is empty) or introduce a custom name (fnVol1F has a value).